src/main/kotlin/matrix/glMatrix.kt

```diff
@@ -0,0 +1,58 @@
+package matrix
+
+import org.khronos.webgl.Float32Array
+import kotlin.js.Math
+
+open class glMatrix {
+    companion object {
+        /**
+         * Common utilities
+         * @module glMatrix
+         */
+        // Configuration Constants
+        var ARRAY_TYPE = Any()
+        val EPSILON = 0.000001
+        val RANDOM = Math.random()
+        /**
+         * Sets the type of array used when creating new vectors and matrices
+         *
+         * @param {Type} type Array type, such as Float32Array or Array
+         */
+        fun setMatrixArrayType(type: Float32Array) {
+            ARRAY_TYPE = arrayOf(type)
+        }
+
+        val degree = Math.PI / 180
+        /**
+         * Convert Degree To Radian
+         *
+         * @param {Number} a Angle in Degrees
+         */
+        fun toRadian(a: Double): Double {
```

Comment:
Delete comments.

naming of a not very well chosen. maybe angleInDegrees instead.